crates/optimism/payload/src/builder.rs

```diff
@@ -205,16 +205,45 @@ where
             requests_root: None,
         };
 
+        // Create an EVM instance
```

Comment:
this is an empty block, so we don't need to execute anything here. we simply return no receipts